### PR TITLE
<TableActionCell/> - Add size to actions

### PR DESCRIPTION
**Pull request by frontned**
### 🔦 Summary

Hey, it would be nice if we could control the action sizes of `<TableActionCell />`. Also we require it in our new functionality:
https://app.zeplin.io/project/5d384bf711d0c52e94d0667c/screen/5f3e0b90381141143e03b31f

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers
- [ ] 👨‍🎨 UX change is approved by the Design System UX
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5911/

File: src/TableActionCell/TableActionCell.js [hunk 1]
```diff
@@ -219,6 +235,9 @@ TableActionCell.propTypes = {
 
   /** Props being passed to the secondary actions' <PopoverMenu/> */
   popoverMenuProps: PropTypes.shape(PopoverMenu.propTypes),
+
+  /** Popover menu button size */
+  popoverMenuButtonSize: PropTypes.oneOf(['small', 'medium']),
```
Comment (EgoziE) [inline src/TableActionCell/TableActionCell.js:240]: Is it really needed to separate between an overall "size" prop and the specific popover "popoverMenuButtonSize"?

I think it's better to have a single global size prop that will make everything the same size.

File: src/TableActionCell/TableActionCell.js [hunk 2]
```diff
@@ -60,6 +62,7 @@ function renderVisibleActions(actions) {
             onClick();
             event.stopPropagation();
           }}
+          size={size || 'medium'}
```
Comment (EgoziE) [inline src/TableActionCell/TableActionCell.js:65]: In all places you use `size={size || 'medium'}` just use the size and set 'medium' under TableActionCell.defaultProps

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/TableActionCell/docs/examples/SmallButtonsExample.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      alwaysShowSecondaryActions
+      primaryAction={{
+        text: 'Edit',
+        skin: 'inverted',
+        onClick: () => window.alert('Primary action was triggered!'),
+      }}
+      secondaryActions={[
+        {
+          text: 'Star',
+          icon: <Star />,
```
Comment (milkyfruit) [inline src/TableActionCell/docs/examples/SmallButtonsExample.js:25]: Small buttons should use small icons. This one should be `<StarSmall />`

File: src/TableActionCell/docs/examples/SmallButtonsExample.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+          onClick: () => window.alert('Star action was triggered.'),
+        },
+        {
+          text: 'Download',
+          icon: <Download />,
```
Comment (milkyfruit) [inline src/TableActionCell/docs/examples/SmallButtonsExample.js:30]: DownloadImportSmall

File: src/TableActionCell/docs/examples/SmallButtonsExample.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+          onClick: () => window.alert('Download action was triggered.'),
+        },
+        {
+          text: 'Duplicate',
+          icon: <Duplicate />,
```
Comment (milkyfruit) [inline src/TableActionCell/docs/examples/SmallButtonsExample.js:35]: DuplicateSmall

File: src/TableActionCell/docs/examples/SmallButtonsExample.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+          onClick: () => window.alert('Duplicate action was triggered.'),
+        },
+        {
+          text: 'Print',
+          icon: <Print />,
```
Comment (milkyfruit) [inline src/TableActionCell/docs/examples/SmallButtonsExample.js:40]: PrintSmall

**Review by milkyfruit — CHANGES_REQUESTED** [review]
The prop looks great, only the example should be updated — small buttons should use small icons.

File: src/TableActionCell/docs/examples/SmallButtonsExample.js (hunk 3 above)
Comment (frontned) [reply src/TableActionCell/docs/examples/SmallButtonsExample.js:25]: There is not `<StarSmall />` in icons

**Pull request merged**